### PR TITLE
Document security officers, the role, and its responsibilities

### LINKED ISSUE #91: Document security officers, the role, and its responsibilities
### Description

Eiffel Community [established](https://github.com/eiffel-community/community/issues/61) and [documented](https://github.com/eiffel-community/community/blob/master/SECURITY.md) security/vulnerability reporting and response process.
This process is overseen by Eiffel Community Security Officers who are appointed by Eiffel Technical Committee.
The role, responsibilities, and term of the Security officers together with the appointed community members must be documented in governance.

### Motivation

The role, its responsibilities, the appointed community members and their term documented in governance for transparency.

### Exemplification

This is a common practice within open source communities.

### Benefits

Increased transparency and communication.

### Possible Drawbacks

N/A

### PR DESCRIPTION
**Pull request by fdegir**
### Applicable Issues

fixes #91 

### Description of the Change

Eiffel Community established and documented security/vulnerability reporting and response process.
This process is overseen by Eiffel Community Security Officers who are appointed by Eiffel Technical Committee.

This PR documents the role, responsibilities, and term of the Security officers together with the appointed community members in governance.

### Alternate Designs

N/A

### Benefits

Increased transparency and communication.

### Possible Drawbacks

N/A

### Sign-off
Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.

Signed-off-by: Fatih Degirmenci fatih.degirmenci@est.tech

### CODE REVIEW

**Review by k-hallen-ericsson — APPROVED** [review]

**Review by e-backmark-ericsson — APPROVED** [review]

**Review by m-linner-ericsson — APPROVED** [review]

**Review by sodero — APPROVED** [review]

**Review by t-persson — APPROVED** [review]

File: GOVERNANCE.md [hunk 1]
```diff
@@ -72,7 +72,24 @@ Eiffel Community Election Officers are the individuals listed below:
...
 
-TBD
+Eiffel Community Security Officers are the community members who are appointed by Eiffel Technical Committee to oversee [Eiffel Community Vulnerability and Security Reporting and Response process](./SECURITY.md).
+
+The term for Security Officers is **one year**.
+
+Responsibilities of Security Officers are
+
+* Be the main contact with regards to community vulnerability and security issues including for the issues reported by users or discovered by the community members
+* Be a member to private community mail list eiffel-community-security@googlegroups.com and monitor it actively
```
Comment (magnusbaeck) [inline GOVERNANCE.md:82]: Nit: "Be a member **of** private ..."

**Review by magnusbaeck — APPROVED** [review]

**Comment by fdegir** [comment]
It looks like we missed submitting this. Can someone submit it please?

**Comment by magnusbaeck** [comment]
> It looks like we missed submitting this. Can someone submit it please?

Sure, let's get this one in. Would it be too much trouble to fix the minor language issue I found first?

**Comment by fdegir** [comment]
> Sure, let's get this one in. Would it be too much trouble to fix the minor language issue I found first?

I directly scrolled down to last part of the page, missing your comment. It's fixed now.
Sorry for this.

**Pull request merged**